### PR TITLE
Sharding and some small tests/improvements

### PR DESCRIPTION
**Pull request by prury**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [x] Test

## Context

- [ ] Related to #7826 

## Screenshot 📸

image

### CODE REVIEW

**Review by roiLeo — COMMENTED** [review]
smoll

File: tests/e2e/profile.spec.ts [hunk 1]
```diff
@@ -81,3 +82,15 @@ test('Profile Interactions', async ({ page, Commands }) => {
     )
   })
 })
+
+test('Verify if there are no Assets on Selected chain', async ({
+  page,
+  Commands,
+}) => {
+  await page.goto(`rmrk/u/${KSM_TEST_ADDRESS_2}`)
```
[the remark this answers is not in the document]
Comment (prury) [reply tests/e2e/profile.spec.ts:90]: changed to a ahp address instead, on a empty new account on the testing seed

File: components/shared/OnRampModal.vue [hunk 2]
```diff
@@ -4,7 +4,7 @@
     class="neo-modal"
     scroll="clip"
     @close="onClose">
-    <div class="w-[unset] lg:w-[25rem]">
+    <div class="w-[unset] lg:w-[25rem]" data-testid="on-ramp-modal">
```
[the remark this answers is not in the document]
Comment (prury) [reply components/shared/OnRampModal.vue]: done

File: components/balance/MultipleBalances.vue [hunk 3]
```diff
@@ -3,13 +3,14 @@
     <div
       v-if="isEmptyBalanceOnAllChains && !isBalanceLoading"
       class="text-xs py-4 flex flex-col items-center">
-      <div class="mb-3 text-center">
+      <div class="mb-3 text-center" data-testid="balances-empty-asset-text">
```
[the remark this answers is not in the document]
Comment (prury) [reply components/balance/MultipleBalances.vue]: indeed, i just waited for the button instead

**Review by preschian — APPROVED** [review]
nice 👍🏻

**Review by roiLeo — APPROVED** [review]

**Pull request merged**